src/game/models/monsters.js

```diff
@@ -500,7 +500,7 @@ t . Fire Emblem
     prizes: {
       exp: 110,
       items: ['Molten Feather'],
-      equips: ['molten_gauntlets', 'flame'],
+      equips: ['flame', 'magma', 'lava', { id: 'molten_gauntlets', weight: 2 }],
```

Comment:
We could continue using strings when there are just strings in the array, and not mix them when using objects, what you think about this?

```
equips: [
  { id: 'flame', weight: 1 } ,
  { id: 'magma', weight: 1 },
  { id: 'lava', weight: 1 },
  { id: 'molten_gauntlets', weight: 2 }
]
```